src/colvarcomp_protein.cpp

```diff
@@ -414,33 +468,10 @@ void colvar::dihedPC::collect_gradients(std::vector<int> const &atom_ids, std::v
 
     for (size_t j = 0; j < theta[i]->atom_groups.size(); j++) {
       cvm::atom_group &ag = *(theta[i]->atom_groups[j]);
-
-      // If necessary, apply inverse rotation to get atomic
-      // gradient in the laboratory frame
-      if (ag.b_rotate) {
-        cvm::rotation const rot_inv = ag.rot.inverse();
-
-        for (size_t k = 0; k < ag.size(); k++) {
-          size_t a = std::lower_bound(atom_ids.begin(), atom_ids.end(),
-                                      ag[k].id) - atom_ids.begin();
-          atomic_gradients[a] += coeff * rot_inv.rotate(ag[k].grad);
-        }
-      } else {
-
-        for (size_t k = 0; k < ag.size(); k++) {
-          size_t a = std::lower_bound(atom_ids.begin(), atom_ids.end(),
-                                      ag[k].id) - atom_ids.begin();
-          atomic_gradients[a] += coeff * ag[k].grad;
-        }
-      }
-      if (ag.is_enabled(f_ag_fitting_group) && ag.is_enabled(f_ag_fit_gradients)) {
-        cvm::atom_group const &fg = *(ag.fitting_group);
-        for (size_t k = 0; k < fg.size(); k++) {
-          size_t a = std::lower_bound(atom_ids.begin(), atom_ids.end(),
-                                      fg[k].id) - atom_ids.begin();
-          // fit gradients are in the unrotated (simulation) frame
-          atomic_gradients[a] += coeff * fg.fit_gradients[k];
-        }
+      for (size_t k = 0; k < ag.size(); k++) {
```

Comment:
Realized treating all the atom group rotation cases was unnecessary for these sub-CVCs!